### PR TITLE
Add redirect document type

### PR DESCRIPTION
**Pull request by DirkDevelopmentFreelance**

### CODE REVIEW

File: api/src/dto/RedirectDto.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    @IsString()
+    @IsNotEmpty()
+    @Expose()
+    fromSlug: string;
+
+    @IsString()
+    @IsOptional()
+    @Expose()
+    toSlug: string;
```
Comment (ivanslabbert) [inline api/src/dto/RedirectDto.ts:20]: Add a '?' to show toSlug is optional

File: api/src/dto/RedirectDto.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    @IsString()
+    @IsOptional()
+    @Expose()
+    toUrl: string;
```
Comment (ivanslabbert) [inline api/src/dto/RedirectDto.ts:25]: add a '?' to show toUrl is optional

**Review by ivanslabbert — CHANGES_REQUESTED** [review]

File: api/src/dto/RedirectDto.ts (hunk 1 above)
Comment (DirkDevelopmentFreelance) [reply api/src/dto/RedirectDto.ts]: Added

**Comment by DirkDevelopmentFreelance** [comment]
Conflicting changes that is too far gone to save so remade this PR.

**Pull request closed without merging**